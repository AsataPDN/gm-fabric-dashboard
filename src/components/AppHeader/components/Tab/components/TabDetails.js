import styled from "styled-components";
import {
  COLOR_ALT_BACKGROUND,
  FONT_SIZE_SM
} from "../../../../../style/styleVariables";
import {
  contrastColor,
  spacingScale
} from "../../../../../style/styleFunctions";

const COLOR_TAB_BACKGROUND_BASE = contrastColor(COLOR_ALT_BACKGROUND, 0.175);

const TabDetails = styled.dl`
  margin: 0 ${spacingScale(2)} ${spacingScale(0.5)};
  font-size: ${FONT_SIZE_SM};
  color: ${contrastColor(COLOR_TAB_BACKGROUND_BASE, 1).string()};
`;

export default TabDetails;
