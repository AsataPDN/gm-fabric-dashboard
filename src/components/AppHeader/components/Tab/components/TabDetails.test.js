import React from "react";
import { shallow } from "enzyme";
import TabDetails from "./TabDetails";

describe("TabDetails", () => {
  it("matches snapshot", () => {
    const aTabDetails = shallow(<TabDetails />);
    expect(aTabDetails).toMatchSnapshot();
  });
});
