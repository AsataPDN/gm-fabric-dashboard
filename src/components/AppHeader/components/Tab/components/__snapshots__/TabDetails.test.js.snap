// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabDetails matches snapshot 1`] = `
.c0 {
  margin: 0 0 4px 24px;
  font-size: 11px;
  color: rgb(255,255,255);
}

<dl
  className="c0"
/>
`;
